src/baldr/accessrestriction.cc

```diff
@@ -1,7 +1,10 @@
-#include "baldr/accessrestriction.h"
```

Comment:
I would strongly vote for slightly different order - valhalla headers first, then 3rd party libs, then stdlib for quite a simple reason - this order prevent accidential "incomplete" headers that works because in place it is included some stdlib/3rd_party include goes first.

And IMHO related header to the current cpp/cc file should **always** go first for exacly the same reason.

Reply:
Well, surprisingly `clang-format` has everything required to do this - https://github.com/valhalla/valhalla/pull/5230

Reply:
@nilsnolde I'm really sorry for the amount of conflicts if #5230 get merged =\

Reply:
Well, I'd figure we merge here first. It should be a lot easier than the other way around I'd figure? You could just run again with the same config, unless you had to do a lot of manual updates in the code?!

Reply:
Aside manual fixes, the flow is extremely simple, assuming that you have `clang-format-11` installed somewhere:

```sh
clang-format-11 -i valhalla/**.h src/**.h src/**.cc test/**.h test/**.cc
```

or with working mason (e.g. on x86_64 linux) just run `./scripts/format.sh`.

Reply:
argh, sadly that does make sense.. alright, I'll do the chore. I'll have to remind myself next time I'm thinking about such a crap work to not start it for a few weeks so it fades away